content/key-topics/climatehealth/heat-story/embed/heat-story-leaflet.js

```diff
@@ -15,13 +15,76 @@ var featureMouseOver = null;
 var legendControl = null;
 var legendPropertiesByLayer = {};
 var layersExclusive = new Set();
+var indicators = [];
+
+const assignGeoRank = (GeoType) => {
+    switch (GeoType) {
+        case 'Citywide':
+            return 0;
+        case 'Borough':
+            return 1;
+        case 'NYCKIDS2017':
+            return 2;
+        case 'NYCKIDS2019':
+            return 2;
+        case 'UHF34':
+            return 3;
+        case 'UHF42':
+            return 4;
+        case 'Subboro':
+            return 5;
+        case 'CD':
+            return 6;
+        case 'CDTA2020':
+            return 7;
+        case 'NTA2010':
+            return 8;
+        case 'NTA2020':
+            return 9;
+    }
+}
+
+// array of (pretty) geotypes in georank order
+
+const geoTypes = [
+    "Citywide",
+    "Borough",
+    "NYCKIDS",
+    "UHF34",
+    "UHF42",
+    "Subboro",
+    "CD",
+    "CDTA",
+    "NTA"
+]
+
+L.TopoJSON = L.GeoJSON.extend({
```

Comment:
I don't think I actually need this anymore. I converted the topo data to geojson, so this should be unnecessary. I will test with it removed.